ui/packages/shared/client/src/google/api/http.ts

```diff
@@ -16,16 +16,19 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
-import type { BinaryWriteOptions } from "@protobuf-ts/runtime";
-import type { IBinaryWriter } from "@protobuf-ts/runtime";
-import { WireType } from "@protobuf-ts/runtime";
-import type { BinaryReadOptions } from "@protobuf-ts/runtime";
-import type { IBinaryReader } from "@protobuf-ts/runtime";
-import { UnknownFieldHandler } from "@protobuf-ts/runtime";
-import type { PartialMessage } from "@protobuf-ts/runtime";
-import { reflectionMergePartial } from "@protobuf-ts/runtime";
-import { MESSAGE_TYPE } from "@protobuf-ts/runtime";
-import { MessageType } from "@protobuf-ts/runtime";
+import {
```

Comment:
I think we should revert the deleted `.prettierignore` file, as we don't need the files in `ui/packages/shared/client` formatted since it's a generated package.

Reply:
So it wasn't moved, I only moved it to be in the `ui/` folder as that's where the prettier config, the `.prettierrc`  file lives.

Reply:
And the now moved file also config for ignoring the client files.